controller/src/main/java/io/pravega/controller/server/bucket/BucketManager.java

```diff
@@ -83,6 +83,15 @@ protected void doStart() {
 
     protected abstract int getBucketCount();
 
+    /**
+     * Get the health status.
+     *
+     * @return true by default.
+     */
+    public boolean isHealthy() {
```

Comment:
This implementation is not needed an essentially is doing the same thing as default method in the interface. Please remove it.

Reply:
Done